src/main/java/racingcar/view/InputView.java

```diff
@@ -1,26 +1,33 @@
-package racingcar.ui;
+package racingcar.view;
+
+import static racingcar.view.OutputCommand.COMMA;
 
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
 
 public class InputView {
-    public static String readCarNames() {
+    private InputView() {
+    }
+
+    public static List<String> readCarNames() {
         System.out.println(OutputCommand.ASK_CAR_NAMES);
         BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
         try {
-            return bufferedReader.readLine();
+            return Arrays.stream(bufferedReader.readLine().split(COMMA)).map(String::trim).toList();
```

Comment:
```suggestion
            return Arrays.stream(bufferedReader.readLine().split(COMMA))
                    .map(String::trim)
                    .toList();
```

[의견] `Stream` 관련 체이닝 메소드를 사용할 때 줄바꿈을 넣어주면 코드 뿐 아니라 IDE 상에서의 가독성도 개선할 수 있어요 :)